homeassistant/components/samsungtv/media_player.py

```diff
@@ -134,7 +155,17 @@ def get_remote(self):
         if self._remote is None:
             # We need to create a new instance to reconnect.
             try:
-                self._remote = SamsungRemote(self._config.copy())
+                if self._config["method"] == "legacy":
```

Comment:
Wouldn't it be better to put the `if` before the `try`? We'll have other exceptions. But on the other hand we have the same "results" (states). Hmmm, unsure...

Reply:
No idea...